opm/autodiff/FullyImplicitBlackoilSolver_impl.hpp

```diff
@@ -286,7 +296,7 @@ namespace detail {
 
             detectNewtonOscillations(residual_norms_history, it, relaxRelTol(), isOscillate, isStagnate);
 
-            if (isOscillate) {
+            if (isOscillate && verbosity_) {
```

Comment:
Nice catch! No it should not be done like this. I just fixed this bug. Many thanks for this.